setup.py

```diff
@@ -3,14 +3,14 @@
 from setuptools import setup, find_packages
 
 base_dir = os.path.dirname(__file__)
-install_requires = [line.rstrip() for line in open(os.path.join(base_dir, 'requirements.txt'))]
+install_requires = [line.rstrip() for line in open(os.path.join(base_dir, 'requirements.in'))]
```

Comment:
This change means that we don't **lock** our dependancies for pip users, allowing them to resolve the dependancies with pip-tools when using this as a library.
But we still have the locked version when using as an application.